engine/crates/gateway-core/src/cache/partial.rs

```diff
@@ -72,6 +72,7 @@ where
                     body,
                     engine::GraphqlOperationAnalyticsAttributes {
                         name: request.operation_name().map(str::to_string),
+                        name_or_generated_one: request.operation_name().map(str::to_string).unwrap_or_default(),
```

Comment:
Do we have a generated name at this stage, or is it ok to default to an empty string?